VideoLocker/src/main/java/org/edx/mobile/view/Router.java

```diff
@@ -46,6 +46,10 @@
     public static final String EXTRA_DISCUSSION_COMMENT = "discussion_comment";
     public static final String EXTRA_DISCUSSION_TOPIC_OBJ = "discussion_topic_obj";
 
+    // FIXME: This would be injected, but dependency injector is not working right (causes runtime exception)
```

Comment:
You are correct. I removed it and switched to using the `@Inject` annotation in the other places.